department-of-veterans-affairs/va.gov-cms
# VACMS-5775: Update meta titles.

## Description
Closes #5775

## Testing done
Behat

## QA steps
- Go to `admin/config/search/metatag`
- Visually verify that there are no instances of `og_image: '[node:field_media:entity:image:entity:url]'` remaining
- Visually verify that default content tag for og image is set under `Content` to `og_image: '[node:field_promo:entity:image:entity:3_2_medium_thumbnail:url]'`
- Visually verify all instances of `title: '[node:title] | [node:field_office] | Veterans Affairs'` have been replaced with `title: '[node:title] | Veterans Affairs'`

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
